scripts/codemods/explicit-barrel-imports.cjs

```diff
@@ -1,39 +1,43 @@
-const fs = require('fs');
 const path = require('path');
 
+const isBareSpecifier = (importPath) => !importPath.startsWith('app/') && /^[^./]/.test(importPath);
```

Comment:
Sorry, it's probably just me not understanding something, but what do we want to filter out exactly with this?

Reply:
We use this to filter out imports of NPM package names. It will return false if the import path starts with `app` or is a relative path (e.g. `./myFile` or `../../somethingElse`).

![image](https://github.com/grafana/grafana/assets/73201/2f8a8e9b-70a1-4480-8954-1f4f534338b7)

By filtering these out we remove the possibility of trying to resolve a path by joining/resolving the file path with npm package names which would cause require.resolve to error out.

Reply:
I see, thanks for the explanation bud 👍